Intermittent integrationt test failure

Just saw this:

```
=================================== FAILURES ===================================

__________________ test_firefox_min_version_updates_correctly __________________



selenium = <selenium.webdriver.firefox.webdriver.WebDriver (session="9129b925-e02c-4ef2-9ef2-e22394e906d2")>

base_url = 'https://nginx/'

fill_overview = <pages.experiment_overview.ExperimentOverview object at 0x7f3c3e44db00>



    @pytest.mark.nondestructive

    def test_firefox_min_version_updates_correctly(selenium, base_url, fill_overview):

        """Test setting a Firefox min version."""

        timeline_pop_form = fill_overview.save_and_continue_btn()

>       assert timeline_pop_form.firefox_min_version == "Versions"



test_timeline_and_population.py:67: 

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

pages/experiment_timeline_and_population.py:86: in firefox_min_version

    element = self.find_element(*self._firefox_min_version_locator)

.tox/integration-test/lib/python3.6/site-packages/pypom/view.py:34: in find_element

    return self.driver_adapter.find_element(strategy, locator)

.tox/integration-test/lib/python3.6/site-packages/pypom/selenium_driver.py:65: in find_element

    return self.driver.find_element(strategy, locator)

.tox/integration-test/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py:978: in find_element

    'value': value})['value']

.tox/integration-test/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py:321: in execute

    self.error_handler.check_response(response)

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 



self = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f3c3e417940>

response = {'status': 404, 'value': '{"value":{"error":"no such element","message":"Unable to locate element: #id_firefox_min_ver...\\nregisterSelf@chrome://marionette/content/listener.js:501:5\\n@chrome://marionette/content/listener.js:1854:1\\n"}}'}



    def check_response(self, response):

        """

        Checks that a JSON response from the WebDriver does not have an error.

    

        :Args:

         - response - The JSON response from the WebDriver server as a dictionary

           object.

    

        :Raises: If the response contains an error message.

        """

        status = response.get('status', None)

        if status is None or status == ErrorCode.SUCCESS:

            return

        value = None

        message = response.get("message", "")

        screen = response.get("screen", "")

        stacktrace = None

        if isinstance(status, int):

            value_json = response.get('value', None)

            if value_json and isinstance(value_json, basestring):

                import json

                try:

                    value = json.loads(value_json)

                    if len(value.keys()) == 1:

                        value = value['value']

                    status = value.get('error', None)

                    if status is None:

                        status = value["status"]

                        message = value["value"]

                        if not isinstance(message, basestring):

                            value = message

                            message = message.get('message')

                    else:

                        message = value.get('message', None)

                except ValueError:

                    pass

    

        exception_class = ErrorInResponseException

        if status in ErrorCode.NO_SUCH_ELEMENT:

            exception_class = NoSuchElementException

        elif status in ErrorCode.NO_SUCH_FRAME:

            exception_class = NoSuchFrameException

        elif status in ErrorCode.NO_SUCH_WINDOW:

            exception_class = NoSuchWindowException

        elif status in ErrorCode.STALE_ELEMENT_REFERENCE:

            exception_class = StaleElementReferenceException

        elif status in ErrorCode.ELEMENT_NOT_VISIBLE:

            exception_class = ElementNotVisibleException

        elif status in ErrorCode.INVALID_ELEMENT_STATE:

            exception_class = InvalidElementStateException

        elif status in ErrorCode.INVALID_SELECTOR \

                or status in ErrorCode.INVALID_XPATH_SELECTOR \

                or status in ErrorCode.INVALID_XPATH_SELECTOR_RETURN_TYPER:

            exception_class = InvalidSelectorException

        elif status in ErrorCode.ELEMENT_IS_NOT_SELECTABLE:

            exception_class = ElementNotSelectableException

        elif status in ErrorCode.ELEMENT_NOT_INTERACTABLE:

            exception_class = ElementNotInteractableException

        elif status in ErrorCode.INVALID_COOKIE_DOMAIN:

            exception_class = InvalidCookieDomainException

        elif status in ErrorCode.UNABLE_TO_SET_COOKIE:

            exception_class = UnableToSetCookieException

        elif status in ErrorCode.TIMEOUT:

            exception_class = TimeoutException

        elif status in ErrorCode.SCRIPT_TIMEOUT:

            exception_class = TimeoutException

        elif status in ErrorCode.UNKNOWN_ERROR:

            exception_class = WebDriverException

        elif status in ErrorCode.UNEXPECTED_ALERT_OPEN:

            exception_class = UnexpectedAlertPresentException

        elif status in ErrorCode.NO_ALERT_OPEN:

            exception_class = NoAlertPresentException

        elif status in ErrorCode.IME_NOT_AVAILABLE:

            exception_class = ImeNotAvailableException

        elif status in ErrorCode.IME_ENGINE_ACTIVATION_FAILED:

            exception_class = ImeActivationFailedException

        elif status in ErrorCode.MOVE_TARGET_OUT_OF_BOUNDS:

            exception_class = MoveTargetOutOfBoundsException

        elif status in ErrorCode.JAVASCRIPT_ERROR:

            exception_class = JavascriptException

        elif status in ErrorCode.SESSION_NOT_CREATED:

            exception_class = SessionNotCreatedException

        elif status in ErrorCode.INVALID_ARGUMENT:

            exception_class = InvalidArgumentException

        elif status in ErrorCode.NO_SUCH_COOKIE:

            exception_class = NoSuchCookieException

        elif status in ErrorCode.UNABLE_TO_CAPTURE_SCREEN:

            exception_class = ScreenshotException

        elif status in ErrorCode.ELEMENT_CLICK_INTERCEPTED:

            exception_class = ElementClickInterceptedException

        elif status in ErrorCode.INSECURE_CERTIFICATE:

            exception_class = InsecureCertificateException

        elif status in ErrorCode.INVALID_COORDINATES:

            exception_class = InvalidCoordinatesException

        elif status in ErrorCode.INVALID_SESSION_ID:

            exception_class = InvalidSessionIdException

        elif status in ErrorCode.UNKNOWN_METHOD:

            exception_class = UnknownMethodException

        else:

            exception_class = WebDriverException

        if value == '' or value is None:

            value = response['value']

        if isinstance(value, basestring):

            if exception_class == ErrorInResponseException:

                raise exception_class(response, value)

            raise exception_class(value)

        if message == "" and 'message' in value:

            message = value['message']

    

        screen = None

        if 'screen' in value:

            screen = value['screen']

    

        stacktrace = None

        if 'stackTrace' in value and value['stackTrace']:

            stacktrace = []

            try:

                for frame in value['stackTrace']:

                    line = self._value_or_default(frame, 'lineNumber', '')

                    file = self._value_or_default(frame, 'fileName', '<anonymous>')

                    if line:

                        file = "%s:%s" % (file, line)

                    meth = self._value_or_default(frame, 'methodName', '<anonymous>')

                    if 'className' in frame:

                        meth = "%s.%s" % (frame['className'], meth)

                    msg = "    at %s (%s)"

                    msg = msg % (meth, file)

                    stacktrace.append(msg)

            except TypeError:

                pass

        if exception_class == ErrorInResponseException:

            raise exception_class(response, message)

        elif exception_class == UnexpectedAlertPresentException:

            alert_text = None

            if 'data' in value:

                alert_text = value['data'].get('text')

            elif 'alert' in value:

                alert_text = value['alert'].get('text')

            raise exception_class(message, screen, stacktrace, alert_text)

>       raise exception_class(message, screen, stacktrace)

E       selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: #id_firefox_min_version



.tox/integration-test/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py:242: NoSuchElementException

------------------------------- pytest-selenium --------------------------------

Driver log: /tmp/pytest-of-seluser/pytest-0/test_firefox_min_version_updat0/driver.log

URL: https://nginx/experiments/this-is-a-test-9sbtf9/edit-timeline-population/

WARNING: Failed to gather log types: Message: HTTP method not allowed

- generated html file: file:///code/app/tests/integration/integration-test.html -

=============== 1 failed, 24 passed, 1 skipped in 231.69 seconds ===============

ERROR: InvocationError for command /code/app/tests/integration/.tox/integration-test/bin/pytest --verify-base-url --base-url https://nginx/ --driver Firefox . -vvv (exited with code 1)

___________________________________ summary ____________________________________

ERROR:   integration-test: commands failed

make: *** [integration_test] Error 1


Exited with code exit status 2

CircleCI received exit code 2
```

https://app.circleci.com/pipelines/github/mozilla/experimenter/2838/workflows/996e8e4d-91c3-4f92-8d98-94caf005d2a1/jobs/8957

--- a/app/tests/integration/pages/experiment_timeline_and_population.py
+++ b/app/tests/integration/pages/experiment_timeline_and_population.py
@@ -30,7 +30,7 @@ class TimelineAndPopulationPage(Base):
         self.wait.until(
             lambda _: self.find_element(
                 By.CSS_SELECTOR, "body.page-edit-timeline-and-population"
-            )
+            ).is_displayed()
         )
         return self
 

